lib/charty/table_adapters.rb

```diff
@@ -0,0 +1,23 @@
+module Charty
+  module TableAdapters
+    @adapters = {}
+
+    def self.register(name, adapter)
+      @adapters[name] = adapter
+    end
+
+    def self.lookup_adapter_maker(data)
```

Comment:
I think method name "find" is more intuitive than "lookup_adapter_maker". How about this?

Reply:
Renamed it to `find_adapter_maker`.